File: MCFCApp/app/src/main/java/com/example/mcfcapp/MainActivity.java
package com.example.mcfcapp;

import androidx.appcompat.app.AppCompatActivity;

import android.os.Bundle;

public class MainActivity extends AppCompatActivity {

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        setTheme(R.style.Theme_MCFCApp);
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_main);

        
    }
}

File: AndroidKotlin/IntentKt/app/src/main/java/com/example/intentkt/MainActivity.kt
package com.example.intentkt

import android.content.Intent
import androidx.appcompat.app.AppCompatActivity
import android.os.Bundle
import kotlinx.android.synthetic.main.activity_main.*

class MainActivity : AppCompatActivity() {
    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_main)

        b_toSub.setOnClickListener() {
            val intent = Intent(this, SubActivity::class.java)
            intent.putExtra("msg", tv_main01.text.toString())
            startActivity(intent)
        }
    }
}